src/bridge/setup.rs

```diff
@@ -30,6 +30,8 @@ const REGISTER_CLIPBOARD_PROVIDER_LUA: &str = r"
         cache_enabled = 0
     }";
 
+const INTRO_MESSAGE: &str = include_str!("../../lua/intro.lua");
```

Comment:
Maybe call this INTRO_MESSAGE_LUA. As is the variable could be construed as the message itself

Reply:
This is fixed now